repo/db/followers_test.go

```diff
@@ -102,15 +120,24 @@ func TestDeleteFollower(t *testing.T) {
 	}
 	defer teardown()
 
-	fdb.Put("abc", []byte("proof"))
+	err = fdb.Put("abc", []byte("proof"))
+	if err != nil {
+		t.Error(err)
+	}
 	err = fdb.Delete("abc")
 	if err != nil {
 		t.Error(err)
 	}
-	stmt, _ := fdb.PrepareQuery("select peerID from followers where peerID=?")
+	stmt, err := fdb.PrepareQuery("select peerID from followers where peerID=?")
+	if err != nil {
+		t.Log(err)
```

Comment:
Setup steps of a test should Fatally fail if the rest of the checks have no chance of succeeding.